src/components/Loader/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,103 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { selectPerson } from '../../../services/selectPerson';
+import { Person } from '../../../types';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const { slug } = useParams();
+
+  useEffect(() => {
+    if (slug) {
+      setSelectedPerson(prev => people.find(p => p.slug === slug) || prev);
+    }
+  }, [slug, people]);
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const currentMother = people.find(p => p.name === person.motherName);
+          const currentFather = people.find(p => p.name === person.fatherName);
+
+          return (
+            <tr
+              data-cy="person"
+              key={person.slug}
+              className={classNames({
+                'has-background-warning': selectedPerson === person,
```

Comment:
The comparison `selectedPerson === person` might not work as expected if `Person` objects are not strictly equal. Consider comparing by a unique identifier like `slug` instead.